core/api/src/app/authentication/login.ts

```diff
@@ -400,3 +401,36 @@ const checkDeviceLoginAttemptPerAppcheckJtiLimits = async (
     rateLimitConfig: RateLimitConfig.deviceAccountCreate,
     keyToConsume: appcheckJti,
   })
+
+export const getAuthTokenFromUserId = async (
```

Comment:
I don't think there is an external use case for it?

this method should probably be in the kratos service. @bodymindarts can confirm

Reply:
Seeing that this is using `kratosAdmin` client directly, which should (correctly) be private to the service boundary I would agree that this needs to be refactored. We are leaking boundary / adapter concerns into the application layer. This code belongs on the service.